.github/workflows/lint_and_format.yml

```diff
@@ -16,12 +16,6 @@ name: Lint and Format
 
 on:
   pull_request:
-    types:
-    - opened
-    - reopened
-    - synchronize
-    branches:
-    - main
```

Comment:
So this is no need?

Reply:
It made builds run only when PR is created to main branch, so this one(or one to development branch) won't build. I think we can skip it